backend/src/openbeheer/zaaktype/tests/e2e/test_list_page.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from playwright.sync_api import Page
+
+from openbeheer.utils.gherkin_e2e import GherkinRunner
+
+
+@pytest.mark.e2e
+def test_list_zaaktypen(page: Page, runner: GherkinRunner):
+    _ = runner
+
+    _.given.user_exists()
+    _.given.objecttypen_service_exists()
+    _.given.selectielijst_service_exists()
+    _.given.ztc_service_exists()
+
+    catalogus = runner.given.catalogus_exists()
+    zaaktypen = _.given.zaaktypen_exist(catalogus, 20)
+
+    # Login
+    _.when.user_open_application(page)
+    _.when.user_logs_in(page, username="johndoe", password="secret")
+    _.then.path_should_be(page, "/OZ/")
+
+    # Open list view
+    _.when.user_selects_catalogus(page, catalogus)
+    _.when.user_navigates_to_zaaktype_list_page(page)
+    _.then.page_should_contain_text(page, "ZAAKTYPE-2025-020")
+    _.then.page_should_contain_text(page, "ZAAKTYPE-2025-019")
+    _.then.page_should_contain_text(page, "ZAAKTYPE-2025-018")
+
+    # Navigate to next page.
+    _.when.user_clicks_on_button(page, "volgende")
+    _.when.user_clicks_on_link(page, "ZAAKTYPE-2025-001")
+
+    # Navigate to detail view
+    _.then.page_should_contain_text(page, "Overzicht", timeout=10000)
```

Comment:
Isn't it better to first assert that we have navigated to the expected page and then check page content?

Reply:
As discussed: auto assert in user_clicks_on_link